contracts/0.4.24/Lido.sol

```diff
@@ -332,6 +325,13 @@ contract Lido is ILido, IsContract, Pausable, AragonApp {
         return _getTotalPooledEther();
     }
 
+    /**
+      * @notice Returns current credentials to withdraw ETH on ETH 2.0 side after the phase 2 is launched
+      */
+    function getWithdrawalCredentials() public view returns (bytes) {
```

Comment:
The same here for the return type. This way we can do without the `bytes <-> bytes32` conversion.

Reply:
OK, Done